fsthttp/backend.go

```diff
@@ -1,11 +1,21 @@
 package fsthttp
 
 import (
+	"errors"
 	"time"
 
 	"github.com/fastly/compute-sdk-go/internal/abi/fastly"
 )
 
+var (
+	// ErrDynamicBackendDisallowed indicates the service is not allowed to
+	// create dynamic backends.
+	ErrDynamicBackendDisallowed = errors.New("dynamic backends not supported for this service")
+
+	// ErrDynamicBackendNameInUse indicates the backend name is already in use.
+	ErrDynamicBackendNameInUse = errors.New("backend name already in use")
```

Comment:
```suggestion
	// ErrBackendNameInUse indicates the backend name is already in use.
	ErrBackendNameInUse = errors.New("backend name already in use")
```

I suggest this because (1) it makes the name shorter and (2) the names have to be unique across all backends, including static ones.